tests/PedantryTest.php

```diff
@@ -37,17 +38,10 @@ public function testMethodsAreOrderedAlphabeticallyByVisibility($className): voi
         );
 
         $getSortValue = function (ReflectionMethod $method) {
-            if ($method->getModifiers() & ReflectionMethod::IS_PRIVATE) {
-                return '2' . $method->getName();
-            }
-
-            if ($method->getModifiers() & ReflectionMethod::IS_PROTECTED) {
-                return '1' . $method->getName();
-            }
+            $prefix = $method->isPrivate() ? '2' : ($method->isProtected() ? '1' : '0');
+            $prefix .= str_contains($method->getDocComment(), '@internal') ? '1' : '0';
```

Comment:
@GromNaN: I saw your comment in [PHPLIB-1238](https://jira.mongodb.org/browse/PHPLIB-1238?focusedCommentId=5704506&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-5704506) but it didn't seem pertinent. The first character here is used to sort on visibility, and the second reports whether `@internal` is used.

Reply:
Ok sounds good